dracut/driver_updates.py

```diff
@@ -411,6 +446,13 @@ def load_drivers(moddict):
     if modules_to_add:
         subprocess.call(["modprobe", "-a"] + modules_to_add)
 
+    # Step 3: Retrigger udev event for net subsystem to force Dracut to load
+    # network settings for new interfaces and for replaced interfaces
+    if net_intfs_unload or pre_remove_intfs != list_net_intfs():
+        log.debug("Retrigger udev events for network interfaces")
+        cmd = ["udevadm", "trigger", "--action=change", "--subsystem-match=net"]
```

Comment:
I don't really understand the trigger here. Shouldn't it be triggered automatically as the new device appears in the system after its module is added? Or is just an *add* action triggered at that time?